packages/eventcatalog/components/SchemaViewer.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo } from 'react';
```

Comment:
We have a `components/Mdx` folder for MDX components (although not perfect), wonder if the component should live inside that, considering its a EventCatalog MDX one? What ya think?

Reply:
Agreed, I'll move it

Reply:
Done